homeassistant/components/fronius/entity.py

```diff
@@ -52,3 +52,8 @@ def _get_entity_value(self) -> Any:
         """
 
         raise NotImplementedError
+
+    def _set_entity_value(self) -> None:
+        """Set the entity value correctly based on the platform."""
+
+        raise NotImplementedError
```

Comment:
using `FroniusEntity(ABC, ...)`
and setting this to 
```py
    @abstractmethod
    def _set_entity_value(self) -> None:
         """Set the entity value correctly based on the platform."""
```
Would show an error on linting, not need to wait for tests to raise `NotImplementedError`.
Same for `get_entity_value`.

Also, I think the `_` prefix of `_FroniusEntity` should be removed as it is used in multiple modules.